<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP Server</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../../../usr/share/icons/gnome/32x32/places/network-server.png</normaloff>../../../../../../usr/share/icons/gnome/32x32/places/network-server.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditUser"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="lineEditPassword"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButtonStart">
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonStop">
      <property name="text">
       <string>Stop</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
